### ISSUE TITLE
Couple of issues while running games after H-encore

### ISSUE THREAD

**Issue by buluayam** [TheOfficialFloW/h-encore#35]
Hi, 

First, big big thanks to TheFlow for the hard work and release - it's a lot of work and appreciation definitely needs to be given. 

Second, just thought of sharing a couple of issues I encountered post H-encore installation, related to games I installed. Probably something to do with the backups themselves or the way I installed them, but thought I would share anyway to contribute to the knowledge in this community. 

Am running Vita on 3.65 with no Enso.

1. Installed a game using Maidump, was straightforward install without additional patches etc, game itself works perfectly fine. However, if I leave the console in sleep mode midway while playing the game, and when I come back to it - I also switch windows in order to connect my bluetooth headset before returning to the game - then the game suddenly cannot detect my vita memory card - none of the saved games are visible and if I try to save a new slot it says "memory card cannot be found". Restarting the console, running Henkaku again and then starting the game up fixes this issue (unless I do above steps again!).

2. Separately, another 2 games I installed (one using Maidump with patches and another using NoNpDrm) - they were initially working when i first installed, a few days later suddenly both are unable to start and both showed the same error message:

Cannot start this application. You must download the application again from [Download List] in [PS Store]. (C0-11136-2)

Incidentally  (don't know if related) this issue happened after issue (1) above occurred.

3. For some reason I can't install Enso, I forget what the error message was. Didn't take note of issue since at the time I decided not to bother. Sorry know this description doesn't help, I mainly wanted to post the first 2 and thought I would mention this anyway. 

Thanks! I am more curious on what the issues could be and to understand more rather than desperately wanting to solve it :)

**Comment by 3750gustavo** [comment]
If you selected the option tô reset tai config on h Encore . You Will pode the plugin necessary for the games to load.
Reconfigure your tai config once more.

Em 6 de jul de 2018 22:52, buluayam <notifications@github.com> escreveu:

Hi,

First, big big thanks to TheFlow for the hard work and release - it's a lot of work and appreciation definitely needs to be given.

Second, just thought of sharing a couple of issues I encountered post H-encore installation, related to games I installed. Probably something to do with the backups themselves or the way I installed them, but thought I would share anyway to contribute to the knowledge in this community.

Am running Vita on 3.65 with no Enso.

  1.  Installed a game using Maidump, was straightforward install without additional patches etc, game itself works perfectly fine. However, if I leave the console in sleep mode midway while playing the game, and when I come back to it - I also switch windows in order to connect my bluetooth headset before returning to the game - then the game suddenly cannot detect my vita memory card - none of the saved games are visible and if I try to save a new slot it says "memory card cannot be found". Restarting the console, running H-encore again and then starting the game up fixes this issue.

  2.  Separately, another 2 games I installed (one using Maidump with patches and another using NoNpDrm) - they were initially working when i first installed, a few days later suddenly both are unable to start and both showed the same error message:

Cannot start this application. You must download the application again from [Download List] in [PS Store]. (C0-11136-2)

Incidentally (don't know if related) this issue happened after issue (1) above occurred.

  1.  For some reason I can't install Enso, I forget what the error message was. Didn't take note of issue since at the time I decided not to bother. Sorry know this description doesn't help, I mainly wanted to post the first 2 and thought I would mention this anyway.

Thanks! I am more curious on what the issues could be and to understand more rather than desperately wanting to solve it :)
